sample/android-app/build.gradle.kts

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright 2020 IceRock MAG Inc. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+plugins {
+    plugin(Deps.Plugins.androidApplication)
+    plugin(Deps.Plugins.kotlinAndroid)
+    plugin(Deps.Plugins.googleServices)
```

Comment:
google services not required for us

Reply:
google services is required for Firebase Crashlytics https://firebase.google.com/docs/crashlytics/get-started?hl=ru&platform=android